### PR TITLE
Correct answer view & Question duplicating

### PR DESCRIPTION
**Pull request by PodviaznikovPavlo**

### CODE REVIEW

**Comment by podviaznikov** [comment]
what is going to happen when the answer is "США" and user types "сша", or "Володимир-Волинський" and user types "володимир-волинський"?

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

oh, i didnt think about that. I will fix it.

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

I have fixed that issue.
Now it works in such way:
i make all answers to lower case. So, if the question is: 'Capital of USA' and answer is 'Washington', and user gives such answers: 'Washington', 'washington', 'WasHinGton', 'WASHINGTON' and so on - that will be correct.

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

also i fixed bug with duplicated questions. it works now.

File: app.js [hunk 1]
```diff
@@ -18,8 +18,16 @@ function getRandomQuestion(){
     return randomQuestion;
 }
 
-function checkTheAnswer(answer){
-    if(question.answer === answer){
+function answerToLowerCase(string){
+    var answer = '';
+    for(var char in string){
```
Comment (podviaznikov) [inline app.js]: no need to do `for` loop. `char` var is of type string. There is no special `char` type in javascript.
So any string has `toLowerCase` method.

**Comment by podviaznikov** [comment]
Idea to convert string to lower case is correct.

When you fix issue with `for` loop I'll accept this PR.

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 
I have removed that method. You can check and merge.

**Pull request merged**